### PR TITLE
Fix type annotation in example, setup elm-verify-examples

### PR DESCRIPTION
**Pull request by jhrcek**
It turns out that this module already works when tested with elm-verify-examples.
It's just that elm-verify-examples doesn't make code example into runnable test case when it doesn't have -`--> expected result` thing (see [Unused functions don't get added to the test](https://github.com/stoeffel/elm-verify-examples#intermediate-definitions)). And since I don't know how to verify that Cmd, just fixing the type annotation and adding "test" script and package.json file so the tests can be run with `yarn test`

### CODE REVIEW

**Comment by jhrcek** [comment]
@turboMaCk this repo doesn't have travis CI enabled yet, so adding initial (probably not working) .travis.yml and going to try to make it work in the followup PRs. Ok to merge?

**Comment by turboMaCk** [comment]
@jhrcek I've turned Travis on for this repo. pls push empty commit as I can't trigger build for this branch from Travis.

**Comment by jhrcek** [comment]
Pushed and elm-verify-examples seems to be running fine (check travis build log).

**Comment by jhrcek** [comment]
I'm just not sure if I should commit yarn.lock or npm's package-lock.json (as the CI is using npm)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: elm
+elm:
+  - elm0.19.0
+
+install:
+  - npm install
```
Comment (turboMaCk) [inline .travis.yml]: yarn install

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+script:
+  - npm run elm-verify-examples
```
Comment (turboMaCk) [inline .travis.yml]: use yarn

**Comment by jhrcek** [comment]
Switched to yarn and CI is :heavy_check_mark:

**Comment by turboMaCk** [comment]
thanks

**Pull request merged**